.github/workflows/deploy-openbridge-static-staging.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy-open-bridge-static-staging
+
+on:
+  push:
+    branches: ["staging"]
```

Comment:
only run CI for staging branch? what about the other branches?

Reply:
We also need an action for deploying to production, right? That triggers off of the "production" branch?

Reply:
I haven't setup the prod infra yet. Will add later.